### PR TITLE
Navigation bar being a function

### PR DESCRIPTION
**Pull request by evanhearne**
# Description

Just putting up for troubleshooting purposes for @laurafitzgerald 

## What

index.html now calls on a function for the navigation bar. Calendar.html is a little messed up, but will be addressed later. All commits from #22 have been added prior to making any changes.

## Why

To troubleshoot index.html styling error at bottom of page.

## How

Changes were made using Sublime Text 3. Jquery tutorials were used to base changes.

## Verification Steps

Clone navigation branch and initialize server using nvm http-server
Head to index.html and troubleshoot bottom of page styling not working correctly.

# Checklist:

- [x] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [ ] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,58 +10,17 @@
...
+    <script>
+    $(document).ready(function() {
+
+        $('#navbar').load('pages/about/navigation.html');
+    });
+</script> 
 </head>
 
 <body>
-    <div class="ui divbg container">
```
[the remark this answers is not in the document]
Comment (evanhearne) [reply index.html]: I'm not entirely sure. Is it linked to the image link code?

**Comment by evanhearne** [comment]
Branched out from calendar branch, regretting it now.

**Comment by laurafitzgerald** [comment]
@evanhearne why are you regretting it?

**Comment by evanhearne** [comment]
Should've done it from master, but I forgot. :)

**Comment by laurafitzgerald** [comment]
so lesson learned right? ;)

**Comment by laurafitzgerald** [comment]
you mentioned that there were two divs with the same id. can you mark which ones you mean?

**Comment by evanhearne** [comment]
Will do that now :)

**Comment by evanhearne** [comment]
I forgot how but index.html line 75 and 91

**Comment by laurafitzgerald** [comment]
Hi @evanhearne Is this pr covered by this one: https://github.com/MountSionCBSSecondary/SchoolWebsite/pull/71 Is that the case?

**Comment by laurafitzgerald** [comment]
thanks @evanhearne closing

**Pull request closed without merging**